user-guide/Advanced_Modules/EPM/Tutorials/EPM_Introduction_Tutorial.md

```diff
@@ -37,7 +37,12 @@ This tutorial consists of the following steps:
    > [!TIP]
    > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
 
-1. Open DataMiner Cube and check whether the following items have been added to your DataMiner Agent:
+1. Connect to the DMS with DataMiner Cube using the argument `ENABLEFEATURE=CPEINTEGRATION`, so that the [CPEIntegration](xref:Overview_of_Soft_Launch_Options#cpeintegration) soft-launch features are enabled.
```

Comment:
I think the note should be made that this is only needed if not using a DaaS system as the DaaS are created with this option enabled. Also, is the ENABLEFEATURE=CPEINTEGRATION and argument that needs to be passed in to cube or just needs to be added to the softlaunch option file?

Reply:
At first I thought this was an option in the soft-launch options file, but upon checking in more detail, I noticed that the CPEIntegration option is mentioned as a Cube-only option (see https://docs.dataminer.services/user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.html#cpeintegration). This means that it doesn't matter if you use DaaS or not, since it's a client-side option. However, if you're not using this option, and everything works fine for you, it could be that the option is not needed for the tutorial after all.

Reply:
Is there a way to set ENABLEFEATURE=CPEINTEGRATION in Cube without the SoftLaunchOptions file? I tested on a server by removing the soft launch option and the topology app went away, so I think the DaaS have the option on by default since I never messed with any of the local Cube options.

Reply:
In that case, the information about the CPEIntegration option on docs must be incorrect, and it's in fact not a Cube-only option. Possibly this got changed at some point and the documentation was not updated. Do you see any difference when the Cube option is or isn't used? I did a quick test myself on my local system, and I get the impression that only the SoftLaunchOptions file now seems to affect which features are displayed.

Reply:
I can't find a way to get the Cube option applied without the softlaunch option, so I think the only way to get it working is with the softlaunch option file.

Reply:
OK, in that case I'll adjust the PR, and I'll also update the information about this soft-launch option, because the current information is clearly incorrect.